### DISCUSSION TITLE
Dynamic DNS for split setups?

### DISCUSSION THREAD

**Discussion by GR0DZ1LLA** [GerryDazoo/Slinger#135] [General]
Has anyone configured any kind of Dynamic DNS for environments where the Server and the Slingbox are in different locations?  Specifically the Media Player is going to x.x.x.x:8080/slingbox to get the Server, then the Server has a config file that points to y.y.y.y (then port forwards through some firewall to a Slingbox)?

Trying to think of the best way to handle this since both networks are consumer ISP plans where the IP's change frequently.

**Comment by GerryDazoo** [comment]
You can put a hostname into the config.ini instead of an IP address if you want. So if you have a dynamic DNS provider you're all set. If not, read the release notes (you have read the release notes?) about using the FInderID and my little server on AWS to provides a similar service but limited to just server connections. Note, this does not help with a slingbox that needs dynamic DNS only the server.
But things might not be as bad as you think. Many ISP don't have a static IP but also they don't change very often. My external IP just changed once in over two years after everything for 100 miles rebooted after hurricane Fiona due to power failures.

**Comment by f520k510** [comment]
Since you have set up a raspberry pi to run the server, there must be a ddns cleint for your ddns service.  Sorry but I am not very familiar with things in this regard.  I run the ddns client on my router.  You can run it on windows or mac, too.
For free ddns service, I use dynu.com
